content/miq_dialogs/miq_provision_ibm_powervs_vm_dialog.yaml

```diff
@@ -87,7 +87,8 @@
         :vm_name:
           :description: Instance Name
           :required_method: :validate_regex
-          :required_regex: !ruby/regexp /^[a-zA-Z]+$/
+          :required_regex: !ruby/regexp /^[a-zA-Z0-9\-_]+$/
+          :required_regex_fail_details: Only letters (no accents), numbers, underscores and dashes are allowed.
```

Comment:
I wonder if there is a better way to say "Only letters (no accents)" since other non-ascii chars like umlauts probably also aren't supported.

Reply:
Wouldn't this match strings like e.g. 'ABC----DEF---123'?

Reply:
Oh interesting, I'm okay matching the text string that they have.  I also think we need to allow cool-sunglasses-emoji as instance names though :)

Reply:
let me figure out how to fulfill this request milord...
-Kuldip